kvmagent/kvmagent/plugins/nfs_primarystorage_plugin.py

```
@@ -344,11 +344,11 @@ def migrate_bits(self, req):
                         rsync_exclude_files.add(filtPath)
                         rsync_excludes = rsync_excludes + " --exclude=%s" % filtPath
 
-            total_size = long(shell.call("rsync -anv %s/ %s %s | grep  -o -P 'total size is \K[\d|,]+'" %
+            total_size = int(shell.call("rsync -anv %s/ %s %s | grep  -o -P 'total size is \K[\d|,]+'" %
```

Ruff (0.8.2)

`shell` may be undefined, or defined from star imports

(F405)